### PR TITLE
MGMT-1486: Image GC support for more than 1K objects

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: internal/imgexpirer/imgexpirer.go [hunk 1]
```diff
@@ -42,14 +42,17 @@ func (m *Manager) ExpirationTask() {
 
 	log.Info("Image expiration monitor woke up, checking for expired images...")
 	// Currently limited to 1000 objects
```
Comment (filanov) [inline internal/imgexpirer/imgexpirer.go:44]: is the comment still relevant?

Comment (avishayt) [reply internal/imgexpirer/imgexpirer.go:44]: No it's not

**Comment by avishayt** [comment]
> How did you test it?

I uploaded 2,500 small files to scality and made sure that it processed all of them.  I would love to add a subsystem test for the GC but don't really know where to begin.

File: internal/imgexpirer/imgexpirer.go (hunk 1 above)
Comment (filanov) [reply internal/imgexpirer/imgexpirer.go:44]: So you can probably delete it.

**Pull request merged**